static/src/javascripts/projects/commercial/modules/messenger/resize.js

```diff
@@ -68,18 +67,6 @@ const init = (register: RegisterListeners) => {
     register('resize', (specs, ret, iframe) => {
         if (iframe && specs) {
             const adSlot = iframe && iframe.closest('.js-ad-slot');
-
-            if (
-                adSlot &&
-                (adSlot.classList.contains('ad-slot--mostpop') ||
-                    adSlot.classList.contains('ad-slot--right') ||
-                    adSlot.classList.contains('ad-slot--offset-right'))
-            ) {
-                // We ignore resize events (sent mainly by apnx)
-                // for the mostpop (Most popular) slot
-                // See https://trello.com/c/TtuGq6Iy
-                return null;
-            }
```

Comment:
Removing this as it previously blocked resizing due to issues with the `adslot` resize event. By removing the resizing of the ad slot, we no longer have this issue.